tests/unit/test_cli.py

```diff
@@ -926,3 +968,23 @@ def test_a_user_can_successfully_save_sboms_merge_results_to_a_file(
             fp.close()
             invoke_expecting_sucess(app, ["merge-sboms", "-o", fp.name, *sbom_files_to_merge])
             assert Path(fp.name).lstat().st_size > 0, "SBOM failed to be written to output file!"
+
+    @pytest.mark.parametrize(
+        "sbom_files_to_merge",
+        [
+            [
+                "./tests/unit/data/sboms/cachi2.bom.spdx.json",
+                "./tests/unit/data/sboms/syft.bom.spdx.json",
+            ],
+        ],
+    )
+    def test_a_user_can_successfully_save_sboms_merge_results_to_a_file_spdx(
```

Comment:
Nitpick: `...to_a_file_in_spdx_format` is human-friendlier than `...to_a_file_spdx`.